astronomer/providers/amazon/aws/example_dags/example_aws_nuke.py

```diff
@@ -0,0 +1,42 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.dummy import DummyOperator
+
+AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID", "**********")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY", "***********")
+AWS_DEFAULT_REGION = os.environ.get("AWS_DEFAULT_REGION", "us-east-2")
+AWS_CONN_ID = os.environ.get("ASTRO_AWS_CONN_ID", "aws_default")
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
```

Comment:
```suggestion
EXECUTION_TIMEOUT_HOURS = int(os.getenv("EXECUTION_TIMEOUT_HOURS", 6))
```
In my opinion makes it more readable since the expected execution timeout is in hours.